src/Compilers/VisualBasic/Portable/Symbols/ObsoleteAttributeHelpers.vb

```diff
@@ -99,12 +127,36 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
             End Select
         End Function
 
+        Private Shared Function IsExperimentalSymbol(namedType As NamedTypeSymbol) As Boolean
+            Return namedType IsNot Nothing AndAlso
+                namedType.Arity = 0 AndAlso
+                namedType.HasNameQualifier("System.Diagnostics.CodeAnalysis", StringComparison.Ordinal) AndAlso
+                namedType.Name.Equals("ExperimentalAttribute", StringComparison.Ordinal)
```

Comment:
:memo: I debated what kind of string comparison to use. On one hand, VB is more relaxed; on the other hand, this type is not defined by the VB language but by the BCL. I ended up opting for the stricter comparison, but feedback is welcome.